backend/services/webhook_service.py

```
@@ -0,0 +1,258 @@
+            await asyncio.gather(*[WebhookService._send(w, event, data) for w in targets])
+        except Exception as exc:
+            logger.error("[Webhook] Error dispatching %s: %s", event, exc)
+        finally:
+            db.close()
+
+    @staticmethod
+    def dispatch_event(event: str, data: Dict[str, Any]) -> None:
+        """Fire-and-forget webhook dispatch. Safe to call from sync context."""
+        try:
+            loop = asyncio.get_running_loop()
+            loop.create_task(WebhookService._dispatch_async(event, data))
```

Ruff (0.15.7)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)